app/src/main/java/com/x8bit/bitwarden/data/vault/datasource/sdk/BitwardenFeatureFlagManager.kt

```diff
@@ -1,11 +0,0 @@
-package com.x8bit.bitwarden.data.vault.datasource.sdk
-
-/**
- * Manages the available feature flags for the Bitwarden application.
- */
-interface BitwardenFeatureFlagManager {
```

Comment:
Any reason why you renamed this?

Reply:
Since the manager can have server-side flags, which may come from a self-hosted environments, I think removing the Bitwarden word from the class makes sense. What do you think?